garasign/gpg-sign/action.yml

```diff
@@ -54,5 +54,5 @@ runs:
           -v $(pwd):$(pwd) \
           -w $(pwd) \
           ${{ inputs.artifactory_registry }}/${{ inputs.artifactory_image }} \
-          /bin/bash -c 'gpgloader && for filename in "${{ inputs.filenames }}"; do gpg --detach-sign --armor --output ${filename}.sig ${filename}; done'
+          /bin/bash -c 'gpgloader && for filename in ${{ inputs.filenames }}; do gpg --detach-sign --armor --output ${filename}.sig ${filename}; done'
```

Comment:
Oh, I was under the assumption that the space separated lists needed to be quoted. This is much better!

Reply:
It would have.  We only support single filenames or a glob pattern now.  But it was surprisingly difficult to generate a space separated list from a directory listing.

Reply:
Ah, I see. Works for me 👍